deploy.sh

```
@@ -1,7 +1,40 @@
-cd $(realpath "$(dirname "${BASH_SOURCE[0]}")") &&
-    gcloud --project pennantchase-256 \
-	   functions deploy --gen2 --region=us-west1 --runtime=python312 \
-	   process_box_score \
-	   --trigger-location=us-west1 --trigger-resource=pc256-box-scores \
-	   --trigger-event=google.storage.object.finalize
+#!/usr/bin/bash
+
+set -o pipefail
+
+shopt -s expand_aliases
+source $HOME/.bash_script_aliases
```

Shellcheck (0.10.0)

[info] Not following: ./.bash_script_aliases was not specified as input (see shellcheck -x).

(SC1091)